test/test_onnx.py

```diff
@@ -482,6 +482,17 @@ def test_keypoint_rcnn(self):
                        dynamic_axes={"images_tensors": [0, 1, 2]},
                        tolerate_small_mismatch=True)
 
+    def test_shufflenet_v2_dynamic_axes(self):
+        model = models.shufflenet_v2_x0_5(pretrained=True)
+        dummy_input = torch.randn(1, 3, 224, 224, requires_grad=True)
+        test_inputs = torch.cat([dummy_input, dummy_input, dummy_input], 0)
```

Comment:
Is there a particular reason to concatenate `dummy_input` to `test_inputs`? Can I use `test_inputs = torch.randn(3, 3, 224, 224, requires_grad=True)`?

Reply:
@fatcat-z any thoughts on this? Is it so that we can compare the output by passing `dummy_input` and `test_inputs` (as they are equal on both dimensions)?